newIDE/app/src/InstancesEditor/index.js

```diff
@@ -1094,18 +1129,39 @@ export default class InstancesEditor extends Component<Props> {
     // $FlowFixMe - JSFunctor is incompatible with Functor
     initialInstances.iterateOverInstances(getInstanceRectangle);
     getInstanceRectangle.delete();
+    return contentAABB;
+  };
+
+  zoomToFitContent = () => {
+    const contentAABB = this.getContentAABB();
     if (contentAABB) this.fitViewToRectangle(contentAABB, { adaptZoom: true });
   };
 
+  getAreaRectangle = (): Rectangle => {
```

Comment:
Nitpicking (this will be helpful when we want to convert this to a functional component, so that there are less things in the interface exposed by it):
```suggestion
  _getAreaRectangle = (): Rectangle => {
```